packages/typespec-test/test/healthinsight/generated/typespec-ts/src/isUnexpected.ts

```diff
@@ -2,31 +2,39 @@
 // Licensed under the MIT license.
 
 import {
-  MatchTrials200Response,
-  MatchTrials202Response,
-  MatchTrialsLogicalResponse,
-  MatchTrialsDefaultResponse,
+  GetJob200Response,
+  GetJobDefaultResponse,
+  CreateJob200Response,
+  CreateJob202Response,
+  CreateJobLogicalResponse,
+  CreateJobDefaultResponse,
 } from "./responses";
 
 const responseMap: Record<string, string[]> = {
+  "GET /trialmatcher/jobs/{jobId}": ["200"],
   "POST /trialmatcher/jobs": ["200", "202"],
   "GET /trialmatcher/jobs": ["200", "202"],
```

Comment:
I wonder if this is expected, in this case, it has a pollingOperation decorator in createJob operation that points to the getJob operation, I don't get why we have a `"GET /trialmatcher/jobs": ["200", "202"],` generated in the expected response map ?

Reply:
Just realize this is probably for querying the resource after the lro succeed.